crates/router/src/core/payments/flows/authorize_flow.rs

```diff
@@ -118,6 +119,14 @@ impl types::PaymentsAuthorizeRouterData {
             _ => Ok(self.clone()),
         }
     }
+
+    fn decide_authentication_type(&mut self) {
```

Comment:
if the merchant passes 3ds auth then how can we change it to no_3ds

Reply:
let say the merchant sends 3DS, and when we are checking the card enrolled we see that the card is not enrolled for 3DS v2, in that case we decide to go with non-3ds flow. we can make this configurable in merchant_connector_meta